src/Controllers/HomeController.cs

```diff
@@ -155,7 +155,7 @@ public async Task<ActionResult> LogInTestUser(StartAppModel startAppModel)
 
             if (startAppModel.AppPathSelection?.Equals("accessmanagement") == true)
             {
-                return Redirect($"/accessmanagement/ui/given-api-delegations/overview");
+                return Redirect($"http://am.ui.local.altinn.cloud/accessmanagement/ui/offered-api-delegations/overview");
```

Comment:
Should probably use the environment variable from the `.env` file here also, in case someone tries to change the config and expect it to work.